src/coreclr/tests/src/JIT/Regression/JitBlue/GitHub_13735/GitHub_13735.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should like this test case go under `:/src/tests/JIT/Regression/JitBlue`? Seems like it is not running in the CI.

Reply:
Thanks! Sent out #45848